library/plugins/graph/graph-publisher/src/main/java/com/cdsap/talaiot/publisher/graph/TaskDependencyGraphPublisher.kt

```diff
@@ -1,11 +1,8 @@
-package com.cdsap.talaiot.publisher
+package com.cdsap.talaiot.publisher.graph
```

Comment:
Let me know if it is okay for this Publisher to have its package changed. I've also kept the same name as before, although I think we could rename it to `GraphPublisher` later on.

Reply:
it's totally ok. The namespace should be detailed, initially I wanted more generic but it's more clear.